test/testdata/packager/apply-all-fixes/a/foo.test.rb

```diff
@@ -0,0 +1,14 @@
+# typed: true
+# selective-apply-code-action: quickfix
+# keep-apply-all-quickfix: true
+
+class Test::A::FooTest
+  B::Foo.new
+# ^^^^^^ error: `B::Foo` resolves but its package is not imported
+# ^^^^^^ apply-code-action: [E] Test Import `B` in package `A`
+# ^^^^^^ apply-code-action: [F] Apply all Sorbet fixes for file
+  B::Foo.new
+# ^^^^^^ error: `B::Foo` resolves but its package is not imported
+# ^^^^^^ apply-code-action: [G] Test Import `B` in package `A`
+# ^^^^^^ apply-code-action: [H] Apply all Sorbet fixes for file
```

Comment:
I'd like to add a test for also referencing some other package and importing that too, but then the assertion that 2 edits don't overlap gets hit, since we'd be inserting both `B` and the other project in the same location.